lib/rubocop.rb

```diff
@@ -715,3 +715,4 @@
   RuboCop::ResultCache.rubocop_required_features = $LOADED_FEATURES - before_us
 end
 RuboCop::AST.rubocop_loaded if RuboCop::AST.respond_to?(:rubocop_loaded)
+require_relative 'rubocop/cop/internal_affairs/useless_restrict_on_send'
```

Comment:
Can you move it to the `lib/rubocop/cop/internal_affairs.rb`?
cf. https://github.com/rubocop/rubocop/pull/10412

Reply:
I'd pushed commit in the middle of my work. Sorry. It has been moved.